### PR TITLE
Submitting form data implemented

### PR DESCRIPTION
**Pull request by avishekjana**
#65

### CODE REVIEW

File: making-api-requests/submit-form-data/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+      <button type="submit">Submit</button>
+    </form>
+  );
+};
+
+export default Form;
+```
+
+Next, we will add event handlers with all input fields to get the updated value in `formData` state.
```
Comment (reenas) [inline making-api-requests/submit-form-data/README.md:41]:
```suggestion
Next, we will add event handlers with all input fields to get the updated value in `formData` state. The updated code will look this:
```

File: making-api-requests/submit-form-data/README.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      <textarea id="message" name="message" value={formData.message} onChange={handleChange} />
+
+      <button type="submit">Submit</button>
+    </form>
+  );
+};
+
+export default Form;
+```
+> Action: Test the show if the form value is getting updated in local state
```
Comment (reenas) [inline making-api-requests/submit-form-data/README.md:78]: @avishekjana This line is not clear. What should they check to know if the form value is getting updated?

File: making-api-requests/submit-form-data/README.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  );
+};
+
+export default Form;
+```
+
+In the `handleSubmit` function, we're making a POST request to the JSONPlaceholder API with the form data using `fetch()`. We're using `async` and `await` to handle the response from the API.
+
+So, we are all set to test it out.
+> Action: Submit the form and Show the output on browser
```
Comment (reenas) [inline making-api-requests/submit-form-data/README.md:139]: @avishekjana What URL should they open in browser to see the output?

File: making-api-requests/submit-form-data/README.md (hunk 2 above)
Comment (avishekjana) [reply making-api-requests/submit-form-data/README.md:78]: Fixed.

File: making-api-requests/submit-form-data/README.md (hunk 3 above)
Comment (avishekjana) [reply making-api-requests/submit-form-data/README.md]: Fixed

**Pull request merged**